parsec/backend/postgresql/organization.py

```diff
@@ -61,15 +61,28 @@
 """
 )
 
-
 _q_get_stats = Q(
     f"""
 SELECT
+    (
+        SELECT ARRAY(
+            SELECT (revoked_on, profile)
+            FROM user_
+            WHERE organization = { q_organization_internal_id("$organization_id") }
+        )
+    )_users,
```

Comment:
What about `users` as result column name ? (given the other `users` result column is not needed anymore, see next comment)